jsapi/src/main.js

```diff
@@ -708,7 +741,7 @@ lux.Map.prototype.addSearch = function(target) {
     'menuClass' : 'lux-search-suggestions',
     'source'    : function(term, suggest) {
       term = term.toLowerCase();
-      fetch(lux.searchUrl + 'limit=5&query=' + term).then(function(resp) {
+      fetch(lux.searchUrl + 'limit=5&layer=Adresse&query=' + term).then(function(resp) {
```

Comment:
I think the integrator should have the possibility to override this setting.
As a default it is Ok, but the integrator should be able to define his own search target. @petzlux @rmichaelis what do you think?